tensorflow/compiler/tf2tensorrt/convert/ops/binary_ops.cc

```diff
@@ -37,11 +37,21 @@ const binaryOperationMap *BinaryOperationMap() {
   return m;
 }
 
+const binaryOperationMap *BinaryBooleanOperationMap() {
+  static auto *const m = new binaryOperationMap({
+      {"LogicalOr", nvinfer1::ElementWiseOperation::kOR},
+      {"LogicalAnd", nvinfer1::ElementWiseOperation::kAND},
+  });
+  return m;
+}
+
 class ConvertBinaryImpl {
  protected:
   ConvertBinaryImpl(const binaryOperationMap *pOperMap) : pOperMap_(pOperMap) {}
 
-  Status ImplValidate(const OpConverterParams &params) {
+  Status ImplValidate(const OpConverterParams &params,
```

Comment:
We use XImpl in the existing code. Let's  do this
s/ImplValidate/ValidateImpl/
s/ImplConvert/ConvertImpl/

Reply:
Done